spec/_layouts/default.html

```diff
@@ -169,12 +169,12 @@ <h1>{{page.title}}</h1>
   		<div class="container">  <!-- container -->
   			<div class="inner-wrapper">
   				<p>
-FHIR IG © {{site.data.info.copyrightyear}} HL7 &amp; Boston Children's Hospital.  
-{{site.data.fhir.packageId}}#{{site.data.fhir.igVer}} based on <a style="color: #81BEF7" href="{{site.data.fhir.path}}">FHIR {{site.data.fhir.version}}</a>. 
+FHIR IG © {{site.data.info.copyrightyear}} HL7 &amp; Boston Children's Hospital.
+{{site.data.fhir.packageId}}#{{site.data.fhir.igVer}} based on <a style="color: #81BEF7" href="{{site.data.fhir.path}}">FHIR {{site.data.fhir.version}}</a>.
 Generated {{site.data.fhir.genDay}}
           <span style="color: #FFFF77"> |
             <a style="color: #81BEF7" target="_blank" href="http://hl7.org/fhir/us/bulkdata/history.cfml">History</a>
-            | <a style="color: #81BEF7" target="_blank" href="https://github.com/HL7/smart-app-launch/issues">Propose a change</a>
+            | <a style="color: #81BEF7" target="_blank" href="https://github.com/HL7/bulk-data/issues">Propose a change</a>
```

Comment:
This needs to point to gforge tracker or better yet an intermediate page on the conference with you that can be redirected to Jira when the time comes.

Reply:
http://hl7.org/fhir-issues is the right permalink

Reply:
http://hl7.org/fhir-issues is the right permalink